app/views/students/index.html.erb

```diff
@@ -1,3 +1,5 @@
+<%= render partial: "errors" %>
```

Comment:
Should we just have this in the `application.html.erb` layout template? Seems like we would want to be able to see error messages on any page.